onnxruntime/core/providers/cpu/quantization/conv_integer.cc

```diff
@@ -155,6 +167,7 @@ Status ConvInteger::Compute(OpKernelContext* context) const {
       gemm_shape.M = static_cast<size_t>(M / conv_attrs_.group);
       gemm_shape.N = static_cast<size_t>(output_image_size);
       gemm_shape.K = static_cast<size_t>(kernel_dim);
+      gemm_shape.BIsSigned = true;
 
```

Comment:
The value of BIsSigned should depends on the data type of input of B. You can not set it to true blindly.

Reply:
FYI. you can use:
gemm_shape.BIsSigned =W->IsDataType<int8_t>()